Maker Launch - Website updates

kelbyhawn opened the pull request:
Update existing marketing pages to announce the launch of the new physical computing/maker unit. 

**Relevant links:** [Asana task](https://app.asana.com/0/1203085657595409/1203483475281335/f) • [Gdoc w/ changes](https://docs.google.com/document/d/1WwXikMdgIDDJ07nQui2xGBAJZGWz6ZUOAApz3TZTlws/edit#bookmark=id.lf7rhz8mw0ss) 
_Note:_ The `/maker` and `/maker/csf-microbit` landing page updates were done in Dropbox

----

## Update screenshots (note: some will look the same)

### code.org/educate/curriculum/high-school
<img width="1006" alt="Screenshot 2023-01-12 at 12 01 54 PM" src="https://user-images.githubusercontent.com/9256643/212169020-1b7fb9a5-2240-4efe-bc70-c7a8692a0631.png">

### code.org/educate/curriculum/middle-school
<img width="994" alt="Screenshot 2023-01-12 at 12 02 43 PM" src="https://user-images.githubusercontent.com/9256643/212169088-dbbc99db-f4a9-41a5-8115-592abefd48e7.png">

### code.org/educate/csp
<img width="1028" alt="Screenshot 2023-01-12 at 12 03 19 PM" src="https://user-images.githubusercontent.com/9256643/212169212-90f33ba7-01bc-4a8c-8d6f-b796bbe9d5d0.png">

### code.org/educate/applab
<img width="1000" alt="Screenshot 2023-01-12 at 12 03 48 PM" src="https://user-images.githubusercontent.com/9256643/212169307-4f5bc583-5511-4086-a296-552e8a9e13e2.png">

### code.org/educate/professional-development-online
<img width="935" alt="Screenshot 2023-01-12 at 12 04 11 PM" src="https://user-images.githubusercontent.com/9256643/212169370-426e2bb6-1dbf-4c17-9a86-ed746ad5e42e.png">

### code.org/educate/curriculum/elementary-school
<img width="1030" alt="Screenshot 2023-01-12 at 12 04 31 PM" src="https://user-images.githubusercontent.com/9256643/212169438-bca3be26-95be-47c1-8c04-ce17d25a5178.png">

### code.org/educate/curriculum/csf 
<img width="1040" alt="Screenshot 2023-01-12 at 12 04 53 PM" src="https://user-images.githubusercontent.com/9256643/212169489-1348cd82-0ca5-4b66-9ed2-2bd0a3968ddc.png">

kelbyhawn commented on pegasus/sites.v3/code.org/public/educate/curriculum/elementary-school.md.erb:
```diff
@@ -49,7 +49,7 @@ How you teach these courses is up to you - as part of your classroom schedule, w
 # Computer Science Connections
 
 <div class="col-50" style="padding-right: 20px; float: left; margin-top: 10px">
-<img src="/images/csc/csc_elementary.png" alt="CS Connections" width="450">
```
Removed this alt text since it's a decorative image

megcrenshaw replied:
Agreed –– sounds good!

kelbyhawn commented on pegasus/sites.v3/code.org/public/educate/csp.md.erb:
```diff
@@ -8,8 +8,8 @@ style_min: true
 <link href="/shared/css/course-blocks.css", type="text/css", rel="stylesheet"></link>
-
 <link href="/css/educate.css" rel="stylesheet">
+<link href="/css/tools.css" rel="stylesheet">
```
This stylesheet is used on this partial: [/pegasus/sites.v3/code.org/views/csp_standalone_units.haml](https://github.com/code-dot-org/code-dot-org/blob/84f0e46206b22aaaf345e543b9a2c77c54b738f5/pegasus/sites.v3/code.org/views/csp_standalone_units.haml)

megcrenshaw commented on pegasus/sites.v3/code.org/views/csp_standalone_units.haml:
```diff
@@ -0,0 +1,39 @@
+              Explore Module
+          .tutorial-info-guide
+            %a{href: "/maker"}
+              More About Maker
```
Nit: is this block supposed to be the same height as the AI block? Usually our "course cards" are the same height, e.g. https://studio.code.org/courses but not sure if it's important

kelbyhawn replied:
I updated this to make them the same height — I did inline styles since these will vary based on the content in them, but I made them both the same height.

megcrenshaw replied:
Excellent!

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/curriculum/elementary-school.md.erb:
```diff
@@ -64,6 +64,25 @@ The curriculum has options for teachers  who’d like to incorporate computer sc
+<a href="/maker/csf-microbit"><button>Learn more</button></a>
+</div>
+</div>
+<div style="clear: both;"></div>
```
Nit –– this code is also part of the CSF page. Is it likely that these will diverge or stay the same? If the same, I'd vote for a partial here, but it's not blocking

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md.erb:
```diff
@@ -42,6 +42,15 @@ Computer Science Discoveries is an introductory course for 6 - 10th grade studen
+[<button>Learn about Maker</button>](/maker)&nbsp;&nbsp;[<button>Explore the Creating Apps for Devices Module</button>](https://studio.code.org/s/devices)&nbsp;&nbsp;
+
+<br>
+<hr>
```
Same nit as above –– this code is also part of the high school page. Is it likely that these will diverge or stay the same? If the same, I'd vote for a partial here, but it's not blocking

kelbyhawn replied:
Excellent idea, and done! (Did this for the other possible partial too.)

megcrenshaw replied:
Wahoo!!

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md.erb:
```diff
@@ -42,6 +42,15 @@ Computer Science Discoveries is an introductory course for 6 - 10th grade studen
+
+Every day, we use devices — computers, phones, and more — that help us accomplish tasks both big and small. When students learn to program these devices, foundational computer science comes alive! In this module, which spans roughly 6 weeks, students will learn the fundamentals of programming devices.
+
+[<button>Learn about Maker</button>](/maker)&nbsp;&nbsp;[<button>Explore the Creating Apps for Devices Module</button>](https://studio.code.org/s/devices)&nbsp;&nbsp;
```
I'm worried about the use of `&nbsp;&nbsp;`, but I think you know more about this than I do. Does this adjust correctly for mobile too? I'm assuming you're doing this to ensure these buttons are on the same line?

kelbyhawn replied:
I copied the same pattern used for the other similar sections on this page that are using Markdown, but I can confirm they look good on mobile. (I agree though, it's not ideal!)

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/professional-development-online.md.erb:
```diff
@@ -95,6 +95,39 @@ No previous experience with coding or computer science is assumed. This is a beg
+
+<ul>
+ <li>Set up class sections and assign the Create Apps with Devices unit to students.</li>
+ <li>Get an introduction to code.or's Create Apps with Devices unit.</li>
```
Typo –– Code.org* with a g (and should it be capitalized?)

kelbyhawn replied:
Good catch! We all missed this on the gdoc 😅

megcrenshaw reviewed (approved):
Once the Code.org typo is fixed, I'm good with this!

I left some small nits and had some questions, but they aren't blocking. This looks great! Thanks for the very helpful screenshots.

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/csp.md.erb:
```diff
@@ -125,7 +125,7 @@ Every lesson plan and activity is tested by a diverse mix of classrooms around t
 [col-50]
 
-<a href="http://www.westcoastanalytics.com/uploads/6/9/6/7/69675515/wca_code_final_report_2018.pdf", target=_"blank"><img src="/images/csp-brown-study.png" style="width:467px"></a>
+<a href="http://www.westcoastanalytics.com/uploads/6/9/6/7/69675515/wca_code_final_report_2018.pdf", target=_"blank"><img src="/images/csp-brown-study.png" style="max-width:467px; width: 100%"></a>
```
Shoot should've seen this earlier! Should this have an empty alt text attribute?

kelbyhawn replied:
I didn't look at anything else on these pages besides the maker stuff (**Edit:** Omg I did look at something else which is evident in this change haha 🤦🏻‍♀️ I noticed this was breaking the container when I was checking the mobile buttons), but I will add these! (This can apply to all the alt tag comments.)

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/applab.haml:
```diff
@@ -162,6 +162,35 @@ video_player: true
+    %tr
+      %td
+        %a{href: CDO.studio_url("/s/devices")}
+          %img{src: "/images/fit-50/animated-examples/circuitplayground.gif"}
```
Same here –– empty alt text attribute?

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/applab.haml:
```diff
@@ -162,6 +162,35 @@ video_player: true
+    %tr
+      %td
+        %a{href: CDO.studio_url("/s/aiml")}
+          %img{src: "/images/fit-50/oceans/mobile-ai.png"}
```
And here –– empty alt text attribute?

megcrenshaw commented on pegasus/sites.v3/code.org/public/educate/curriculum/csf.md.erb:
```diff
@@ -96,6 +96,13 @@ Express courses are designed for situations where the teacher is not able to pla
+<br>
+<br>
+
+<%= view :maker_microbit_info %>
```
Love this! It'll be so easy to update 🥳

megcrenshaw reviewed (approved):
Thanks for fixing stuff so quickly! This looks great

dmcavoy commented:
Thank you Kelby for all of this work! It all looks so amazing. I'm realizing that on Monday we decided on "Unit" instead of "Module". If you have a minute to replace module with unit great otherwise it will be ok the way it is

kelbyhawn commented:
> Thank you Kelby for all of this work! It all looks so amazing. I'm realizing that on Monday we decided on "Unit" instead of "Module". If you have a minute to replace module with unit great otherwise it will be ok the way it is

Just updated `module > unit` and `Micro:bit > micro:bit` — _note:_ I didn't change any instances of module if it wasn't related to Maker/micro:bit. I replaced the screenshots in this PR and will also update in the gdoc.
